packages/commonwealth/client/scripts/views/components/FloatingActionButton/FloatingActionButton.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+import './FloatingActionButton.scss';
+
+type FloatingActionButtonProps = {
+  onClick: () => void;
+  children: React.ReactNode;
+};
+
+export const FloatingActionButton = ({
+  children,
+  onClick,
+}: FloatingActionButtonProps) => {
+  return (
+    <div className="FloatingActionButton" onClick={onClick}>
```

Comment:
plz make it an interactive element ex: `button` instead of `div`

Reply:
Thanks ! Fixed!